src/coreComponents/python/CMakeLists.txt

```diff
@@ -4,19 +4,19 @@ if ( Python3_EXECUTABLE )
      # Select the version of python to target
      if( ENABLE_PYGEOSX )
           set( PYTHON_POST_EXECUTABLE ${CMAKE_BINARY_DIR}/lib/PYGEOSX/bin/python CACHE PATH "" FORCE )
```

Comment:
@TotoGaz - I've moved the check for the virtualenv package to the pygeosx block here, since it isn't a firm requirement for the environment setup script anymore.

Reply:
Thx. @sframba as you are using `pygeosx` a lot, is it OK for you?